documentation/src/main/asciidoc/topics/proc_converting_data_on_demand.adoc

```diff
@@ -0,0 +1,22 @@
+[id='converting-data-on-demand_{context}']
+= Converting data on demand
```

Comment:
As this just uses the embedded APIs, maybe this section should just be "Embedded Data Conversion"